server/src/test/java/com/broadcom/lsp/cobol/service/delegates/references/DocumentOccurrencesTest.java

```diff
@@ -32,68 +32,47 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
 
 /** This class checks the expected behavior of Go to Definition request resolving. */
-class DocumentOccurrencesTest extends ConfigurableTest {
-  private static final String TEXT =
-      "       Identification Division. \n"
-          + "       Program-id.    ProgramId.\n"
-          + "       Data Division.\n"
-          + "       Working-Storage Section.\n"
-          + "       01   outer1. \n"
-          + "        02   inner1      PIC 9(4) Binary. \n"
-          + "        02   inner2      PIC X(10).\n"
-          + "       01   outer2.\n"
-          + "        02   inner1      PIC 9(4) Binary. \n"
-          + "        02   inner2      PIC X(10).\n"
-          + "       02   Str          PIC 9(4) Binary. \n"
-          + "       Procedure Division.\n"
-          + "       000-Main-Logic.\n"
-          + "           Perform 100-Test.\n"
-          + "           Stop Run.\n"
-          + "       100-Test.\n"
-          + "           Move inner1 of outer1 to Str.\n"
-          + "           Move inner2 of outer1 to Str.\n"
-          + "           Move inner1 of outer2 to Str.\n"
-          + "           Move inner2 of outer2 to Str.\n"
-          + "       COPY CPYBK1.\n"
-          + "       COPY CPYBK2.\n"
-          + "       COPY CPYBK1.\n"
-          + "       End program ProgramId.";
+class DocumentOccurrencesTest extends MockTextDocumentService {
```

Comment:
This test should be converted to a proper unit test for the `Occurrences` delegate framework and check the actual behavior of the classes.